samples/dictionary.sample.yml

```diff
@@ -0,0 +1,763 @@
+# Key to Entry Fields
+#
+# Each object represents a *dictionary entry*.
+#
+# category:                                         # The lexical category for the entry. This is displayed to the end user as-is, but it is also used by the dictionary app for ...? For Plains Cree, the value of this field is whatever is in Arok's `\ps` field in the Toolbox file. For now, each entry may only have one lexical category.
+# FST:                                              # Object containing information for the FST.
+#   lemma:                                          # The lemma used by the FST, in the proto-orthography.
```

Comment:
Should we include here an element with the full FST-analysis for the entry head, for completeness' sake?